packages/store-sync/src/createStoreSync.ts

```diff
@@ -145,11 +149,12 @@ export async function createStoreSync<TConfig extends StoreConfig = StoreConfig>
         await storageAdapter({ blockNumber, logs: chunk });
         onProgress?.({
           step: SyncStep.SNAPSHOT,
-          percentage: ((i + chunk.length) / chunks.length) * 100,
+          percentage: Math.floor((i / chunks.length) * 100),
```

Comment:
I am intentionally not rounding here so frontends can decide how much precision they want to render. Let’s remove this diff.